obspy/core/tests/test_utcdatetime.py

```diff
@@ -1484,7 +1484,7 @@ def test_strftime_with_years_less_than_1900(self):
         t = UTCDateTime(1888, 1, 2, 1, 39, 37)
         self.assertEqual(t.strftime('%Y-%m-%d'), '1888-01-02')
         t = UTCDateTime(998, 11, 9, 1, 39, 37)
-        self.assertEqual(t.strftime('%Y-%m-%d'), '0998-11-09')
+        self.assertEqual(t.strftime('%Y-%m-%d'), '998-11-09')
```

Comment:
@d-chambers IIRC we had some code to make sure %Y is 4 characters always. It's long time ago so not sure

Reply:
Right you are, I had seen the python 3 check and assumed it wasn't needed any more. I re-added it and reverted the test.